src/app/events/schemas/SpeakerSchema.ts

```diff
@@ -2,14 +2,17 @@ import { z } from 'zod'
 
 import { ImagePropsSchema } from '@/schemas/ImagePropsSchema'
 
-const speakerSchema = z.object({
+const speakersListSchema = z.object({
   name: z.string(),
   title: z.string(),
   company: z.string(),
   linkedin: z.string().url(),
   image: ImagePropsSchema,
 })
 
-export const SpeakersSchema = z.array(speakerSchema)
-
-export type Speaker = z.infer<typeof speakerSchema>
+export const SpeakersSchema = z.object({
+  kicker: z.string().optional().default('Speakers'),
+  title: z.string().optional().default('Speakers'),
```

Comment:
```suggestion
  kicker: z.string().default('Speakers').optional(),
  title: z.string().default('Speakers').optional(),
```

Reply:
I think this one needs to be `.optional().default('Speakers')`, otherwise it generates type `string | undefined`